README.md

```diff
@@ -46,7 +46,7 @@ npm install -g gitlab-ci-local
 
 ### Linux
 
-Download and put binary in `/usr/local/bin`
+Download and put binary in `/usr/local/bin`:
```

Comment:
@alexislefebvre What is the purpose of this colon ?

Reply:
This is common before an explanation: https://en.wikipedia.org/wiki/Colon_(punctuation)#Usage_in_English

But the other items doesn't have colons, I can remove it here.